docs/getting-started.md

```diff
@@ -3,33 +3,111 @@ layout: docs
 title: Getting started
 weight: 0
 ---
-Welcome! These instructions show you how to integrate Staticman with your site.
+Welcome! These instructions walk you through deploying your own instance of Staticman and connecting it to your static site.
 
-These steps relate to the hosted version of the platform; if you wish to host it yourself on your infrastructure, check the [GitHub repository](https://github.com/eduardoboucas/staticman).
+## Step 1. Git provider authentication
 
-## Step 1. Add Staticman to your repository
+Staticman currently supports the following git providers:
 
-I will need push access to the repositories you want me to commit the files to. In GitHub, go to your repository **Settings** page, navigate to the **Collaborators** tab and add the username **staticmanapp**.
+- GitHub
+- GitLab
 
-![Step 1](/assets/images/get-started/step1.png)
+In order to use Staticman, the repository for your static site must be hosted on one of these providers. The Staticman service needs to authenticate with the git provider to commit files and handle pull requests. This step will walk you through obtaining the necessary credentials.
 
-At this point, the invitation will be pending. In order for me to accept it, you need to open:
+### **If using GitLab to host the static site repo:**
 
-`https://api.staticman.net/v2/connect/{your GitHub username}/{your repository name}`
+[Create a personal access token](https://docs.gitlab.com/ee/user/profile/personal_access_tokens.html#creating-a-personal-access-token) with the following scopes
 
-If all goes well, you should receive a message saying **OK!** and we're good to go.
+- `read_repository`: Necessary to read the Staticman site config
+- `write_repository`: Necessary to merge pull requests
 
-## Step 2. Create a configuration file
+### **If using GitHub to host the static site repo:**
 
-I will look for a `staticman.yml` file in the root of the repository, where various configuration parameters will be defined.
+#### **Option 1. Authenticate as a GitHub application**
 
-You can use the [sample config file](https://github.com/eduardoboucas/staticman/blob/master/staticman.sample.yml) as a starting point and check the [available configuration parameters](/docs/configuration) for more information.
+This is the recommended way to authenticate with GitHub. This method will give the most control over what Staticman can and can't access.
```

Comment:
even though this (v1) is the recommended approach, almost all the examples later on in Getting Started are based on `v3`, which may potentially confuse user. My suggestion: either use v3 examples and make the option 2 recommended, or continue using option 1 as recommended and give examples based on v1.

Reply:
@LordAmit  To clarify this further, v3 is the current API version and is thus the recommended path to use *except in the case of the webhook endpoint*. It's on my plate to bring the webhook endpoint up to speed with everything else, but for now it's stuck using /v1.

Again, this is only for the webhook callback. All other API calls should be using v3 currently.